### PR TITLE
iOS is now thread-safe.

### PR DESCRIPTION
**Pull request by tcard**

### CODE REVIEW

File: content/realtime/usage.textile [hunk 1]
```diff
@@ -150,80 +150,7 @@ blang[objc,swift].
...
-          } else {
-              self.ablyQueue.sync(execute: operation)
-          }
-      }
-  }
-  ```
-
-  We're working on improving this situation to be more developer-friendly and less error-prone.
-
+  **Note: The iOS client libraries are thread-safe.** Check "the README":https://github.com/ably/ably-ios#thread-safety for details.
```
Comment (mattheworiordan) [inline content/realtime/usage.textile]: This is good. Given we have older 1.* versions that are not thread-safe, do you think it would make sense to say it was thread-safe since version `1.0.8`?

File: content/realtime/versions/v0.8/usage.textile [hunk 2]
```diff
@@ -129,79 +129,7 @@ blang[objc,swift].
...
-              operation()
-          } else {
-              self.ablyQueue.sync(execute: operation)
-          }
-      }
-  }
-  ```
-
-  We're working on improving this situation to be more developer-friendly and less error-prone.
+  **Note: The iOS client libraries are thread-safe.** Check "the README":https://github.com/ably/ably-ios#thread-safety for details.
```
Comment (mattheworiordan) [inline content/realtime/versions/v0.8/usage.textile]: Why did you change this?  `v0.8` isn't thread-safe right?

**Comment by tcard** [comment]
@mattheworiordan PTAL

**Comment by mattheworiordan** [comment]
Thanks @tcard
